projects/step-frontend/src/lib/modules/parameter/components/parameter-list/parameter-list.component.html

```diff
@@ -77,11 +77,19 @@
         <step-parameters-key [context]="element" />
       </td>
     </ng-container>
-    <ng-container matColumnDef="value" stepActivityCol="Value" stepSearchCol>
+    <ng-container matColumnDef="value.value" stepActivityCol="Value" stepSearchCol>
       <th mat-header-cell *matHeaderCellDef mat-sort-header>
         <step-drag-column-caption>Value</step-drag-column-caption>
       </th>
-      <td mat-cell *matCellDef="let element">{{ element.value }}</td>
+      <td mat-cell *matCellDef="let element">
+        @if (element.value.dynamic) {
+          <step-icon class="icon-small" name="zap-circle" matTooltip="dynamic expression" matTooltipPosition="above">
+          </step-icon>
+          {{ element.value.expression }}
+        } @else {
+          {{ element.value.value }}
```

Comment:
After saving the parameter (if we parse parameter.value to string), the retrieved value is a strange format: ""{\"0\":\"{\",\"1\":\"\\\"\",\"2\":\"0\",\"3\"...